docs/pages/machine-workload-identity/machine-workload-identity.mdx

```diff
@@ -1,6 +1,79 @@
 ---
-title: "Machine & Workload Identity"
+title: "Teleport Machine & Workload Identity: Integrated Security"
 description: "Provides information on the Machine & Workload Identity product."
 ---
 
-<DocCardList />
+Teleport Machine & Workload Identity offers two complementary types of identity for non-human entities in your infrastructure:
+
+**Zero Trust Access for machines**: Enables machines (like CI/CD pipelines) to securely authenticate with your Teleport cluster to access protected resources and configure the cluster itself.
+**Flexible Workload Identities**: Issues short-lived cryptographic identities to workloads, compatible with the SPIFFE standard, enabling secure workload-to-workload communication and third-party API authentication.
+
+## Secure service-to-service authentication
+
+Establish a root certificate authority within your Teleport cluster that issues short-lived JWTs and X509 certificates to workloads. These identities ([SPIFFE](../machine-workload-identity/workload-identity/spiffe.mdx) Verifiable Identity Documents or SVIDs) contain the workload's identity encoded as a URI (SPIFFE ID).
+
+Key benefits:
+
+- Eliminates long-lived shared secrets
+- Establishes a universal form of identity for workloads
+- Simplifies infrastructure by reducing authentication methods
+
+The tbot agent manages identity requests and renewals, authenticating to the Teleport cluster using supported join methods. Workloads receive identities either through filesystem/Kubernetes secrets or via the SPIFFE Workload API.
+
+## Zero Trust Access for machines
+
+Teleport provides machines with an identity ("bot") that can authenticate to the Teleport cluster. Bots are similar to human users with access controlled by roles and activities recorded in audit logs.
+
+Bots authenticate using join tokens that specify which bot user they grant access to and what proof (join method) is needed. Each tbot client connection creates a server-side Bot Instance to track installations over time.
+
+## Integrated use cases
+
+Workload Identity and Machine ID can work together to create a comprehensive security model. Machines can securely access resources while workloads communicate securely with each other and external services, all managed through Teleport's unified access plane.
```

Comment:
I think we need to replace "Workload Identity" and "Machine ID" here with the names used in the rest of the document.